forms-web-app/src/views/submit-appeal/grounds-appeal.njk

```diff
@@ -0,0 +1,151 @@
+              rejected. You should not include sensitive information (also called ‘special category information’) in
+              your statement. If you do, you will be asked to resubmit and this will delay your appeal (or invalidate
+              it, if you do not resubmit in time).</p>
+
+            <p>Sensitive information refers to:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>Representations from children</li>
+              <li>Information relating to children</li>
+              <li>Information relating to health</li>
+              <li>Information relating to crime</li>
+              <li>Any information relating to an individual’s:
+                <ol type="i">
+                  <li>race</li>
+                  <li>ethnic origin</li>
+                  <li>politics</li>
+                  <li>religion</li>
+                  <li>trade union membership</li>
+                  <li>genetics</li>
+                  <li>physical characteristics</li>
+                  <li>sex life</li>
+                  <li>sexual orientation</li>
+                </ol>
+              </li>
+            </ul>
+          </div>
+        </details>
+        <div class="govuk-inset-text">File size should be no more than 1.5GB</div>
+        <div class="govuk-form-group">
+          <label class="govuk-heading-m" for="appeal-statement">
+            Upload a file
+          </label>
+          <input value="{{ data['appeal-statement'] }}" class="govuk-file-upload" id="appeal-statement"
+                 name="appeal-statement" type="file">
+        </div>
+
+        <details class="govuk-details" data-module="govuk-details">
+          <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">
+              Help with file formats
+            </span>
+          </summary>
+          <div class="govuk-details__text">
+            Please upload your statement in one of the following formats:
+            <ul class="govuk-list--bullets">
+              <li>PDF &ndash; <strong>.pdf</strong></li>
+              <li>Microsoft Word &ndash; <strong>.doc</strong> or <strong>.docx</strong></li>
+              <li>TIF &ndash; <strong>.tif</strong> or <strong>.tiff</strong></li>
+              <li>JPEG &ndash; <strong>.jpg</strong> or <strong>.jpeg</strong></li>
+              <li>PNG &ndash; <strong>.png</strong></li>
+            </ul>
+          </div>
+        </details>
+
+        <div class="govuk-form-group">
+          <div class="govuk-checkboxes">
+            <div class="govuk-checkboxes__item">
+              <input class="govuk-checkboxes__input" id="sense-check" name="sense-check" value="on" type="checkbox">
+              <label class="govuk-label govuk-checkboxes__label" for="sense-check">
+                I confirm that I have not included sensitive information in my appeal statement.
```

Comment:
Please could we use the gov uk component here?

https://design-system.service.gov.uk/components/checkboxes/